pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,27 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.error("The app #{app.name} does not have a default external group.")
+        UI.error("Please make sure to pass group names to the `:groups` option.")
+        return
+      end
+
+      default_external_group.add_tester!(tester)
```

Comment:
If `config[:groups]` is set, it should pass the return statement on the next line and then go into adding the tester to all the groups.